fe/fe-core/src/main/java/com/starrocks/task/AgentTaskQueue.java

```diff
@@ -341,4 +341,11 @@ public static synchronized List<Object> getSamplesForMemoryTracker() {
         }
         return result;
     }
+
+    public static synchronized void failForLeaderTransfer() {
+        Map<Long, Map<Long, AgentTask>> createTasks = tasks.column(TTaskType.CREATE);
```

Comment:
only need to handle `CREATE` type tasks?

Reply:
Yes, other tasks do not affect the execution of SQL.